bundles/net.tourbook/plugin.xml

```diff
@@ -1317,7 +1339,14 @@
       <command
          id             ="command.net.tourbook.views.TourWaypointView"
          name           ="%View_Name_Waypoint"
-         defaultHandler ="net.tourbook.ui.views.ActionHandler_OpenView_TourWaypoint"/>
+         defaultHandler ="net.tourbook.ui.views.ActionHandler_OpenView_TourWaypoint"
+      />
```

Comment:
This code was reformatted, all other commands have a different formatting which is much cleaner

Reply:
Fixed